urllib3/util/__init__.py

```diff
@@ -24,6 +24,10 @@
     split_first,
     Url,
 )
+from .selectors import (
+    wait_for_read,
+    wait_for_write
+)
```

Comment:
I think that having the same interface as traditional selectors is valuable. I will move the `wait_for_read` and `wait_for_write` to a different file.  Any suggestions on where these should end up? :)